papaparse.js

```diff
@@ -1508,6 +1508,10 @@ License: MIT
 				if (duplicateHeaders) {
 					var editedInput = input.split(newline);
 					editedInput[0] = Array.from(headerMap).join(delim);
+					// If we expanded the input due to duplicate headers then reduce cursor
+					// by the amount we expanded the input.
+					// This is needed for keeping leftover aggregate in parseChunk.
```

Comment:
This comment isn't super clear to me. I wonder if this would be better?

```
// If we expand the input due to duplicate headers, we need to record the difference
// so that we can adjust the cursor accordingly in `meta.cursor` value of the `parse` result.
//
// This is because the consumers of this method (e.g. ChunkStreamer) use the resulting 
// `cursor` value to know how much of the input was consumed by the parser and are not 
// aware of the parser implementation details for handling duplicate headers.
```

Reply:
Changed it to 
```
// If we change the size of the input due to duplicate headers
// or header renaming from transformHeader, then we need to
// record the difference so that we can adjust the cursor accordingly
// in `meta.cursor` value of the `parse` result.
// This is because the consumers of this method (e.g. ChunkStreamer)
// use the resulting `cursor` value to know how much of the input was
// consumed by the parser and are not aware of the parser implementation
// details for handling duplicate headers.
```

A little verbose. We'll see if the maintainer is ok with it.